docs/how-to/distributed-training.txt

```diff
@@ -83,30 +83,49 @@ S3. In this example, the S3 bucket name is configured via a field
           s3.download_file(s3_bucket, data_file, filepath)
       return download_directory
 
+.. _dtrain-scheduling:
 
 Scheduling Behavior
 -------------------
 
-To use distributed training, ``slots_per_trial`` must be set to a multiple
-of the number of GPUs per machine. For example, if you have a cluster of machines with
-8 GPUs each, you should set ``slots_per_trial`` to a multiple of 8, such as 8, 16, or 24.
-This ensures that the full network and interconnect bandwidth are available to
-multi-GPU workloads and results in better performance.
+The Determined master takes care of scheduling distributed training jobs
+automatically, ensuring that all of the compute resources required for a job are
+available before the job itself is launched. Users should be aware of the
+following details about scheduler behavior when using distributed training:
+
+- When using Docker host-mode networking (e.g., when
```

Comment:
non-blocking: I would re-order these points to be in the inverse order of what they are now, since I expect most customers would be running in docker networking and this top point isn't applicable to them.